app/src/main/java/net/gaast/giggity/Schedule.java

```diff
@@ -512,16 +512,29 @@ private void loadJson(BufferedReader in) {
 				String startTimeS = event.getString("start_time");
 				String endTimeS = event.getString("end_time");
 				Date startTime, endTime;
-				startTime = new Date(getTimeInMillis(startTimeS));
-				endTime = new Date(getTimeInMillis(endTimeS));
-				Schedule.Item item = new Schedule.Item(uid, title, startTime, endTime);
+
+                startTimeS = startTimeS.replace('T', ' ');
```

Comment:
You're doing this because T in the format string above doesn't work?

But I *think* you can make the format string something like "yyyy-MM-dd'T'HH:mm:ss" so this substitution won't be necessary.